src/socketHandler/gameHandler.js

```diff
@@ -33,7 +33,7 @@ const startGame = async function (socket, io) {
 
   addSubround(lobby.lobbyid);
 
-  const gameData = startRound(1, 3);
+  const gameData = startRound(1, 3, lobby.maxRounds);
```

Comment:
the second parameter of start round (3) is false if we want to play a game with more than 3 people.

Reply:
Was introduced in #44, but since that PR is already merged I will fix it here